pycarol/connectors.py

```diff
@@ -321,10 +326,7 @@ def get_dm_mappings(self, connector_id=None, connector_name=None, staging_name=N
             url = "v1/connectors/mappings/all"
 
         else:
-            if connector_name:
-                connector_id = self.get_by_name(connector_name)['mdmId']
-            else:
-                assert connector_id
+            connector_id = connector_id if connector_id else self.get_by_name(connector_name)['mdmId']
```

Comment:
on Delete operation there is a check and a deprecation message. 
I think both should have the same behaviour right?

Reply:
checking the APIs it is mandatory connecotr_id in this one.. I will add the check